libs/sdk-backend-bear/src/backend/workspace/attributes/elements/limitingAfmFactory.ts

```diff
@@ -59,7 +61,7 @@ export class LimitingAfmFactory {
                     displayForm: toBearRef(this.displayFormRef),
                 },
             ],
-            filters: filtersPart,
+            filters: filtersPart || relativeDateFilter,
```

Comment:
in case when both types of filters are provided, we should combine them together